db/models.py

```diff
@@ -1 +1,53 @@
 from django.db import models
+from django.utils import timezone
+
+
+class Race(models.Model):
+    RACES = [
+        ("Elf", "Elf"),
```

Comment:
in choices first element in tuple should beexactly how you are going to pass it - in our json, all races are lower case